packages/app-desktop/gui/NoteEditor/utils/resourceHandling.ts

```
@@ -162,7 +162,33 @@ const processImagesInPastedHtml = async (html: string) => {
+					// Strip whitespace before decoding, then save as a Joplin resource
+					// so Turndown's outerHTML (used for images with width/height) gets
+					// a short file:// URL instead of 200KB of base64.
+					const cleanSrc = imageSrc.replace(/\s/g, '');
+					const dataUrlMatch = cleanSrc.match(/^data:([^;]+);base64,(.+)$/);
+					if (dataUrlMatch) {
+						const mimeType = dataUrlMatch[1];
+						const base64Data = dataUrlMatch[2];
+						const fileExt = mimeUtils.toFileExtension(mimeType) || 'bin';
+						const filePath = `${Setting.value('tempDir')}/${md5(Date.now() + Math.random())}.${fileExt}`;
+						try {
+              await shim.fsDriver().writeFile(filePath, base64Data, "base64");
```

ESLint

[error] Expected indentation of 7 tabs but found 14 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 7 tabs but found 14 spaces.

(indent)